### ISSUE TITLE
[Feature request] Adding Computational log likelihood in torch CRF Library

### ISSUE THREAD

**Issue by 506610466** [skorch-dev/skorch#711]
`
```
from torchcrf import CRF
class BiLSTMCRF(nn.Module):
    def __init__(self):
        super(BiLSTMCRF, self).__init__()
        self.embedding = nn.Embedding(len(dataset.text_itos), config.embedding_dim)
        self.lstm = nn.LSTM(config.embedding_dim, config.hidden_size // 2,
                            num_layers=config.num_layers, batch_first=True,
                            bidirectional=True, dropout=config.dropout)

        self.fc = nn.Linear(config.hidden_size, len(dataset.tag_itos))
        self.crf = CRF(num_tags=len(dataset.tag_itos))
   def log_likelihood(self,emissions):
          return -self.crf(emissions, tags, mask=mask)
```
`

This loss function can not use CRF in skorch

**Comment by BenjaminBossan** [comment]
Sorry, I don't get your question. Maybe you could post a more complete code example, including the error message you got?

**Comment by BenjaminBossan** [comment]
Since there was no response for a month, I will close this issue. If you can provide more background information, feel free to re-open.
